packages/data/src/world/mm/great_bay_temple.yml

```diff
@@ -94,9 +94,9 @@
     "Great Bay Temple Pot Compass Room Surface 2": "true"
     "Great Bay Temple Pot Compass Room Surface 3": "true"
     "Great Bay Temple Pot Compass Room Surface 4": "true"
-    "Great Bay Temple Pot Compass Room Water 1": "underwater_walking || short_hook_anywhere"
-    "Great Bay Temple Pot Compass Room Water 2": "underwater_walking || short_hook_anywhere"
-    "Great Bay Temple Pot Compass Room Water 3": "underwater_walking || short_hook_anywhere"
+    "Great Bay Temple Pot Compass Room Water 1": "underwater_walking || short_hook_anywhere || (can_dive_small && (has_weapon_range || has_bombchu))"
+    "Great Bay Temple Pot Compass Room Water 2": "underwater_walking || short_hook_anywhere || (can_dive_small && (has_weapon_range || has_bombchu))"
+    "Great Bay Temple Pot Compass Room Water 3": "underwater_walking || short_hook_anywhere || (can_dive_small && (has_weapon_range || has_bombchu))"
```

Comment:
these should check specifically for arrows or hook since deku bubbles just splat on the water's surface.

Reply:
No they dont.